lib/wickedbase.pm

```diff
@@ -225,6 +244,21 @@ sub ping_with_timeout {
     return 0;
 }
 
+=head2 setup_static
+
+  setup_static()
+
+Setup the first interface from static ifcfg file.
+
+=cut
+sub setup_static {
+    my ($self) = @_;
+    my $iface = iface();
```

Comment:
this means yet another `script_output` while we already **know** interface in all places where we calling setup_static. let's get it as input param and not involve unneeded interaction with VM

Reply:
but we will need to call iface() from outside anyways, so same number of script_output, isn't it?